### ISSUE TITLE
REST API: /loadtracks return friendlyexception

### ISSUE THREAD

**Issue by zaza7** [lavalink-devs/Lavalink#163]
When using the REST API endpoint /loadtracks and it fails due to copyright block, the response given back is the following
 ```
{
    "playlistInfo": {},
    "loadType": "LOAD_FAILED",
    "tracks": []
}
```
is it possible to also have the reason why the track failed to load? For example "This video contains content from *whomever*, who has blocked it in your country on copyright grounds."

**Comment by freyacodes** [comment]
That should be possible. The exception is provided here: https://github.com/Frederikam/Lavalink/blob/master/LavalinkServer/src/main/java/lavalink/server/player/AudioLoader.java#L101

**Comment by schnapster** [comment]
Implemented by #174 (Thanks @Devoxin!)
